clients/venice-client/src/main/java/com/linkedin/venice/fastclient/StatsAvroGenericStoreClient.java

```diff
@@ -34,6 +36,8 @@ public class StatsAvroGenericStoreClient<K, V> extends DelegatingAvroStoreClient
 
   private final int maxAllowedKeyCntInBatchGetReq;
   private final boolean useStreamingBatchGetAsDefault;
+  private final RedundantExceptionFilter redundantExceptionFilter =
+      new RedundantExceptionFilter(RedundantExceptionFilter.DEFAULT_BITSET_SIZE, TimeUnit.SECONDS.toMillis(1));
```

Comment:
Can we change it to one log per minute?

Reply:
changed it to default which is one log per minute.